### DISCUSSION TITLE
About versions...

### DISCUSSION THREAD

**Discussion by Bunker56** [PhlexPlexico/G5API#242] [Q&A]
Hi!..
0.13-updates
extra_stats
As far as I understand, these branches will be designed to interact with the new Get 5  v0.13?

**Comment by PhlexPlexico** [comment]
In G5WS, that branch is only there to ensure compatibility when it gets released.

The extra stats branch in the API is to track `OnDeath` calls with additional stats. Depending on motivation I may take a crack at swapping over to just Get5 logs, but PRs are always welcome for it.
